lib/bike_brigade/delivery.ex

```diff
@@ -170,6 +170,17 @@ defmodule BikeBrigade.Delivery do
 
   alias BikeBrigade.Delivery.CampaignRider
 
+  def get_campaign_rider_by_id(campaign_id, rider_id) do
+    CampaignRider |> Repo.get_by(campaign_id: campaign_id, rider_id: rider_id)
+  end
+
+  def update_campaign_rider(%CampaignRider{} = rider, attrs) do
+    rider
+    |> CampaignRider.changeset(attrs)
+    |> Repo.update()
+    |> broadcast(:campaign_updated)
```

Comment:
Move closer to create_campaign rider
Note we're broadcasting `:campaign_updated` should be `:campaign_rider_udated`

Update the handle_info to do the same as `:campaign_rider_created`